src/hydro/srcterms/tabular_cooling.hpp

```diff
@@ -150,14 +157,16 @@ class TabularCooling {
     if (log_temp < log_temp_start) {
       // Below table - return 0 or first entry?
       // log_lambda = log_lambdas(0);
-      // TODO(forrestglines):Currently, no cooling is implemented above the
-      // table. This behavior could be generalized via templates
-      return 0;
+      // TODO(forrestglines):Currently, zero cooling is use for temperatures
+      // below the table. This behavior could be generalized via templates
+      // return 0;
+      log_lambda = log_lambdas(0); // HACK - to get it to cool to floor
```

Comment:
Related to the comment above. We should discuss next week during the call what we physically want (in terms of floors, choice of cooling table, etc).